index.html

```diff
@@ -788,347 +818,442 @@
-              <p>
-                Pixel layout: planar yuv-channels
-              </p>
-              <p>
-                Data type: 8-bit unsigned integer
-              </p>
-            </dd>
-            <dt>
-              YUV420SP_NV12
-            </dt>
-            <dd>
-              <p>
-                Channel order: Y, U, V
-              </p>
-              <p>
-                Channel size: full y-channel, quarter uv-channels
-              </p>
-              <p>
-                Pixel layout: planar y-channel, interleaving uv-channels
-              </p>
-              <p>
-                Data type: 8-bit unsigned integer
-              </p>
-            </dd>
-            <dt>
-              YUV420SP_NV21
-            </dt>
-            <dd>
-              <p>
-                Channel order: Y, V, U
-              </p>
-              <p>
-                Channel size: full y-channel, quarter uv-channels
-              </p>
-              <p>
-                Pixel layout: planar y-channel, interleaving vu-channels
-              </p>
-              <p>
-                Data type: 8-bit unsigned integer
-              </p>
-            </dd>
-            <dt>
-              HSV
-            </dt>
-            <dd>
-              <p>
-                Channel order: H, S, V
-              </p>
-              <p>
-                Channel size: full hsv-channels
-              </p>
-              <p>
-                Pixel layout: interleaving hsv-channels
-              </p>
-              <p>
-                Data type: 8-bit unsigned integer
-              </p>
-            </dd>
-            <dt>
-              Lab
-            </dt>
-            <dd>
-              <p>
-                Channel order: l, a, b
-              </p>
-              <p>
-                Channel size: full lab-channels
-              </p>
-              <p>
-                Pixel layout: interleaving lab-channels
-              </p>
-              <p>
-                Data type: 8-bit unsigned integer
-              </p>
-            </dd>
-            <dt>
-              DEPTH
-            </dt>
-            <dd>
-              <p>
-                Channel order: DEPTH
-              </p>
-              <p>
-                Channel size: full depth-channel
-              </p>
-              <p>
-                Pixel layout: planar depth-channel
-              </p>
-              <p>
-                Data type: 16-bit unsigned integer
-              </p>
-            </dd>
-          </dl>
+          <pre class="idl">
+            enum ImageFormat {
+                "RGBA32",
+                "BGRA32",
+                "RGB24",
+                "BGR24",
+                "GRAY8",
+                "YUV444P",
+                "YUV422P",
+                "YUV420P",
+                "YUV420SP_NV12",
+                "YUV420SP_NV21",
+                "HSV",
+                "Lab",
+                "DEPTH",
+                /* empty string */ ""
```

Comment:
Correct. Since the return type of the `findOptimalFormat()` method is `ImageFormat` and the method can return an empty string, the enum should have the empty string.

This behaves similarly to e.g. the following enums: https://html.spec.whatwg.org/#canplaytyperesult https://html.spec.whatwg.org/#autocompleteerrorreason